packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-document-model/document-type.esm.t

```diff
@@ -0,0 +1,6 @@
+---
+to: "<%= rootDir %>/<%= paramCaseDocumentType %>/gen/document-type.ts"
+force: true
+---
+
```

Comment:
export document type as a named export from a single file to avoid circular imports